### PR TITLE
Errorpages#100

### PR DESCRIPTION
**Pull request by Idontker**
created a basic error-page-component to handle errors listed in #100 
Additionally created tests for the new component:
- might be good example to see how to mock services
- 🥳 useless goal of 100% test coverage archived 🥇

### CODE REVIEW

**Comment by Idontker** [comment]
forgot to tag the RM
Hi @Jean28518 🥇

**Comment by steve-237** [comment]
everything is clear in my opinion!

File: src/digitalIdentity-frontend/src/app/error-pages/error-page/error-page.component.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      '⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣰⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀\n' +
+      '⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠰⡷⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠓⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀\n' +
+      '⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠐⠑⠳⠳⡷⡷⡷⡷⡷⠷⠳⠳⠑⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀\n' +
+      '⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀\n' +
+      '⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀\n' +
+      '⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀\n' +
+      '⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀'
+    );
+  }
+}
```
Comment (steve-237) [inline src/digitalIdentity-frontend/src/app/error-pages/error-page/error-page.component.ts:53]: Nice!

**Review by steve-237 — COMMENTED** [review]
I approve

**Pull request merged**